src/vim.h

```diff
@@ -439,8 +439,11 @@ typedef unsigned int u8char_T;	// int is 32 bits or more
  * The vimlong_T has sizeof(vimlong_T) >= 2 * sizeof(int).
  * One use is simple handling of overflow in int calculations.
  */
+#ifdef HAVE_NO_LONG_LONG
+typedef long      vimlong_T;
+#else
 typedef long long vimlong_T;
-
+#endif
```

Comment:
```suggestion
#endif

```
Please insert an empty line after `#endif` here.